README.md

```diff
@@ -160,34 +159,37 @@ As a first step you'll need to:
    - The generated key value will be displayed when you click the **Add** button. Copy the generated value for use in the steps later.
    - You'll need this key later in your code's configuration files. This key value will not be displayed again, and is not retrievable by any other means, so make sure to note it from the Azure portal before navigating to any other screen or blade.
 
-#### Configure the webApp app (WebApp_Python_Flask_Authentication_MyOrg) to use your app registration
+#### Configure the webApp app (python-flask-webapp-auth-my-tenant) to use your app registration
 
 Open the project in your IDE to configure the code.
 
 > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
-1. Open the `config.py` file
-1. Find the app key `default-value-enter-your-tenant-id-here` and replace the existing value with your Azure AD tenant ID.
-1. Find the app key `default-value-enter-your-client-id-here` and replace the existing value with the application ID (clientId) of the `WebApp_Python_Flask_Authentication_MyOrg` application copied from the Azure portal.
-1. Find the app key `default-value-enter-your-client-secret-here` and replace the existing value with the key you saved during the creation of the `WebApp_Python_Flask_Authentication_MyOrg` app, in the Azure portal.
+1. Open the `aad.config.json` file
+1. Find the string `default-value-enter-your-tenant-id-here` and replace the existing value with your Azure AD tenant ID.
```

Comment:
>default-value-enter-your-tenant-id-here [](start = 20, length = 39)

someone mixed up and added "default-value-" in one of the config files and we keep copying this everywhere? Remove "default-value-" . Check the code gen as well please.